website/src/constants/index.tsx

```diff
@@ -27,6 +27,7 @@ export const API = {
 export enum VIEW_PORT {
   MOBILE_BREAKPOINT = 760,
   TABLET_BREAKPOINT = 768,
+  LAPTOP_BREAKPOINT = 960,
 };
```

Comment:
how are three different breakpoints? why are we not using the ones we have defined in the theme?

Reply:
These breakpoints are used to change the elements with respective of the design, inorder to get the correct design we are using these breakpoints.

Reply:
The theme defined breakpoints are material-ui default,